include/openssl/service_indicator.h

```diff
@@ -0,0 +1,40 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef AWSLC_HEADER_SERVICE_INDICATOR_H
+#define AWSLC_HEADER_SERVICE_INDICATOR_H
+
+#if defined(__cplusplus)
+extern "C" {
+#endif
+
+OPENSSL_EXPORT uint64_t awslc_fips_service_indicator_get_counter(void);
+OPENSSL_EXPORT uint32_t awslc_fips_service_indicator_get_serviceID(void);
+
+#if defined(AWSLC_FIPS)
+#define IS_FIPS_APPROVED_CALL_SERVICE(approved, func)                       \
```

Comment:
why not include serviceid call in the macro scope? It hides more details and is easier to use.